rj_gameplay/rj_gameplay/tactic/nmark_tactic.py

```diff
@@ -98,7 +98,7 @@ def create_request(self, **kwargs) -> role.RoleRequest:
 
     def get_requests(
         self, world_state: rc.WorldState, props
-    ) -> List[tactic.RoleRequests]:
+    ) -> tactic.RoleRequests:
```

Comment:
Error: rj_gameplay/rj_gameplay/tactic/nmark_tactic.py:99: error: Return type "List[Dict[SkillEntry[Any], List[RoleRequest]]]" of "get_requests" incompatible with return type "Dict[SkillEntry[Any], List[RoleRequest]]" in supertype "ITactic"
+
Error: rj_gameplay/rj_gameplay/tactic/nmark_tactic.py:119: error: Incompatible return value type (got "Dict[SkillEntry[Any], List[RoleRequest]]", expected "List[Dict[SkillEntry[Any], List[RoleRequest]]]")
-> There doesn't seem to be any reason hinted in the code to return a list of dictionaries, so list hint was removed.